Examples/BasicTutorial/AdvancedFeatures/07-hoc/android/app/build.gradle

```diff
@@ -0,0 +1,211 @@
+apply plugin: "com.android.application"
+
+import com.android.build.OutputFile
+
+/**
+ * The react.gradle file registers a task for each build variant (e.g. bundleDebugJsAndAssets
+ * and bundleReleaseJsAndAssets).
+ * These basically call `react-native bundle` with the correct arguments during the Android build
```

Comment:
I guess this sentence would be 'These are basically called' or 'They basically call'

Reply:
This is auto generated comment by react native.